cfn-guard-rulegen-lambda/Makefile

```diff
@@ -121,3 +121,9 @@ invoke: run err
 
 uninstall:
 	aws lambda delete-function --function-name $(project_name)
+
+install-cdk: build
+	cd cdk; npm install; npm run build; cdk bootstrap; cdk deploy -c function-name=$(project_name) CfnGuardRulegenStack --require-approval=never
```

Comment:
Wouldn't it be better to ask for approval here?

Reply:
Wanted to keep the install and uninstall workflow with CDK similar to the existing ones. The lambda execution role through IAM is already a prerequisite.